src/lib/structures/HelpPaginatedMessage.ts

```diff
@@ -0,0 +1,240 @@
+import { LanguageKeys } from '#lib/i18n/languageKeys';
+import { minutes } from '#utils/common';
+import {
+	isGuildBasedChannel,
+	isMessageButtonInteraction,
+	PaginatedMessageAction,
+	PaginatedMessageOptions,
+	PaginatedMessagePage
+} from '@sapphire/discord.js-utilities';
+import type { TFunction } from '@sapphire/plugin-i18next';
+import { isFunction } from '@sapphire/utilities';
+import { Constants, Message, MessageActionRow, MessageButton, MessageOptions, MessageSelectMenu, User, WebhookEditMessageOptions } from 'discord.js';
+import { SkyraPaginatedMessage } from './SkyraPaginatedMessage';
+
+export class HelpPaginatedMessage extends SkyraPaginatedMessage {
+	private language: TFunction;
+
+	public constructor(language: TFunction, options: PaginatedMessageOptions = {}) {
+		super(options);
+		this.setIdle(minutes(10));
+		this.language = language;
+
+		this.actions = new Map<string, HelpPaginatedMessageAction>([
```

Comment:
Why aren't you using setActions

Reply:
Good question. Somehow I didn't realise it was there. Just a sec.